src/main/java/org/springframework/data/couchbase/core/mapping/CouchbaseDocumentSimpleTypes.java

```diff
@@ -0,0 +1,23 @@
+package org.springframework.data.couchbase.core.mapping;
+
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+
+import org.springframework.data.mapping.model.SimpleTypeHolder;
+
+public class CouchbaseDocumentSimpleTypes {
```

Comment:
Rename the class to CouchbaseSimpleTypes as it doesn't seem specific CouchbaseDocument?

Reply:
Done.